Extrapolator.cpp

```diff
@@ -546,7 +554,14 @@ void Extrapolator::send_all_announcements(uint32_t asn,
             if (ann.second.priority < 200) {
                 continue;
             }
-            
+           
+            // Full path generation
+            auto cur_path = ann.second.as_path;
+            // Handles appending after origin
+            if (cur_path.back() != asn) {
```

Comment:
This causes an invalid_read in the Extrapolator.cpp unit tests when the cur_path is empty. Change to
```suggestion
            if (cur_path.size() == 0 || cur_path.back() != asn) {
```

Reply:
The path should never be empty, is this a compiler issue?